soccer/src/soccer/planning/planner/intercept_planner.hpp

```diff
@@ -9,7 +9,7 @@ namespace planning {
  * Whether this means moving and stopping in the path of the ball
  * or completely driving through and "slapping" the ball.
  *
- * Mostly used for the goalie / defenders to block shots
+ * Mostly used for the goalie to block shots (w/ a target point of 0,0).
```

Comment:
Is it really always going to have a target of 0,0? Even if we are the other team?

Reply:
you are right, should say "w a target point of our goal", I'll fix that